insights/config/specs.py

```diff
@@ -118,6 +118,7 @@
                                     SimpleFileSpec("sos_commands/foreman/foreman-debug/etc/foreman-proxy/settings.yml")]),
     "foreman_proxy.log"         : First([SimpleFileSpec("var/log/foreman-proxy/proxy.log", large_content=True),
                                     SimpleFileSpec("sos_commands/foreman/foreman-debug/var/log/foreman-proxy/proxy.log", large_content=True)]),
+    "freeradius"                : NoneGroup([PatternSpec(r"etc/raddb/(.+\.conf|(mods-enabled|policy\.d|sites-enabled)/.+)")]),  # /etc/raddb/*.conf + /etc/raddb/mod-enabled/* + /etc/raddb/policy.d/* + /etc/raddb/sites-enabled/*
```

Comment:
`insights/config/specs.py` doesn't exist in Insights Core 3.x. All specs (now known as datasources) should have a RegistryPoint entry in the Specs class found in `insights/specs/__init__.py`. Datasources for files that should be collected directly from a host, insights archive, or sos report (and share a relative path in all those contexts) are defined in insights/specs/default.py. If the path is different in insights archives or sos reports, put definitions in `insights/specs/insights_archive.py` and `insights/specs/sos_archive.py` respectively. If a datasource will collect multiple files, set `multi_output=True` on its `RegistryPoint`. If the data should be raw and not split line by line in advance, set `raw=True` on its `RegistryPoint` and pass `kind=RawFileProvider` in the definition. Have a look at the [Rule Migration Guide](https://github.com/RedHatInsights/insights-core/blob/master/notebooks/Rule%20Migration%20Guide.ipynb) and the [Datasource Registry Notebook](https://github.com/RedHatInsights/insights-core/blob/master/notebooks/Datasource%20Registry.ipynb) for details on how datasources work in 3.x.

Also, make sure all the files returned by a single datasource make sense for a single parser (they should share the same syntax and semantics, the first of which the parser interprets and the later of which the parser exposes with a higher level API than the raw lines).